packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## `6.33.2`
+
+- Added new feature to manage zos-logs. [Issue 1104](https://github.com/zowe/zowe-cli/issues/1104)
```

Comment:
For changelog headers, please use `## Recent Changes` since we want to avoid having to change this manually at the time this PR is merged.
Same applies to any other changelogs in this PR 😋

Reply:
@zFernand0 sorry I did not get this one, I followed the previous format like:
```
## `6.33.1`

- Bugfix: Fixed capitalization of handler paths for `zowe files rename ds` and `zowe files rename dsm` commands.
```
So could you please advise a little bit more on how to write this? Thank you!

Reply:
The idea behind doing ...
```
## Recent Changes

- Enhancement: Added new feature to zos-logs. [#1104](link)
```

...is so that your PR does not have to be changing constantly if/when there is an enhancement merged before yours.

For example:
Say that you use `## 6.34.0` but there is another PR with an enhancement that get merged before yours, now you will have to use the "next available version" `## 6.35.0`.
Instead of doing that, we have automated the changelog headers so that whenever a new version gets published, the `## Recent Changes` is replaced by the `## <M.m.p>` header.

Reply:
Got it, thank you very much for the explanation @zFernand0 , could you please review again?